src/Compilers/CSharp/Portable/Binder/Semantics/Conversions/UserDefinedImplicitConversions.cs

```diff
@@ -609,6 +609,9 @@ private static bool IsEncompassingImplicitConversionKind(ConversionKind kind)
                 case ConversionKind.ImplicitTupleLiteral:
                 case ConversionKind.ImplicitTuple:
                 case ConversionKind.ImplicitThrow:
+
+                // Added for C# 8.
+                case ConversionKind.ImplicitNew:
                     return true;
```

Comment:
Do we have a test that exercises this branch?

Reply:
Looks like it's not reachable for ImplicitThrow as well. My understanding is that since both new and throw are convertible to *any* type, there's no conversion left for these to encompass. Is that correct?